### ISSUE TITLE
metadata extractor content guidelines

### ISSUE THREAD

**Issue by mih** [datalad/datalad-metalad#85]
The demo in #84 had

```
              "content": {
                "@id": "datalad:SHA1-s4--a8fdc205a9f19cc1c7507a60c4f01b13d11d7fd0",
                "comment": "test-implementation",
                "content_byte_size": 108,
                "intra_dataset_path": "mike",
                "path": "/tmp/mike/mike",
                "type": "file"
              },
```

while I am uncertain about the whole content structure, I am not commenting on it here, as it is likely just a demo. However, it is worth pointing out that any absolute path information like `/tmp/mike/mike` should never be part of any extractor output, as it could not possibly generalize.

**Comment by christian-monch** [comment]
You are right, it is a demo, and the absolute path was just inserted to simplify some verification tasks. There will be no absolute paths in the final ("real") metadata results

**Comment by christian-monch** [comment]
Addressed in PR #122
